src/quart_injector/scope.py

```diff
@@ -1,9 +1,4 @@
-"""
-Request :class:`~injector.Scope`.
-"""
-
 import typing
```

Comment:
```suggestion
"""
 Request :class:`~injector.Scope`.
 """

 import typing
```

why remove the comment and spacing? this will upset the linters.